ee/codegen/src/__test__/__snapshots__/project.test.ts.snap

```diff
@@ -10,7 +11,7 @@ if __name__ != "__main__":
     raise Exception("This file is not meant to be imported")
 
 
-runner = SandboxRunner(workflow=Workflow, inputs=[Inputs(input="foo"), Inputs(input="bar"), Inputs(input="hello")])
+runner = SandboxRunner(workflow=Workflow, inputs=[Inputs(input="foo", chat=[ChatMessageRequest(role="USER", text="foo")]), Inputs(input="bar", chat=[ChatMessageRequest(role="USER", content=StringChatMessageContentRequest(value="bar"))]), Inputs(input="hello", chat=[ChatMessageRequest(role="USER", content=[StringChatMessageContentRequest(value="hello")])]),])
```

Comment:
why is this not newlining after introducing the comma?

Reply:
My guess is you're hitting this case: https://github.com/vellum-ai/vellum-python-sdks/blob/b0c07f84f6694cf6a344bd0b7e261a4e616e9270/ee/codegen/src/generators/base-persisted-file.ts#L87